src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return  new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(99));
```

Comment:
Consider using a single instance of `ColorSupplier` and `Random` by creating them as instance variables, rather than creating new instances every time the `getRandomBall()` method is called. This can improve performance and reduce unnecessary object creation. 🚀

Reply:
Magic numbers should be constants with proper naming